plugins/rapid7_insightidr/help.md

```diff
@@ -504,10 +505,10 @@ Example input:
 
 ##### Output
 
-|Name|Type|Required|Description|
-|----|----|--------|-----------|
-|investigations|[]investigation|True|A list of found investigations|
-|metadata|investigation_metadata|True|The pagination parameters used to generate this page result|
+|Name|Type|Required|Description|Example|
+|----|----|--------|-----------|-------|
+|investigations|[]investigation|True|A list of found investigations|None|
+|metadata|investigation_metadata|True|The pagination parameters used to generate this page result|None|
```

Comment:
Could you please fill out those output examples in plugin spec? 😄

Reply:
I have went through and added any examples that were missing